### PR TITLE
made air-to-vac conversion param when reading linelist

### PR DESCRIPTION
**Pull request by alexander-wise**

### CODE REVIEW

File: src/pipeline/prep_line_list.jl [hunk 1]
```diff
@@ -2,7 +2,7 @@
 """ prepare_line_list( linelist_fn, spectra, pipeline; Δv_to_avoid_tellurics, v_center_to_avoid_tellurics )
 linelist_fn is the full path to the line list file
 """
```
Comment (eford) [inline src/pipeline/prep_line_list.jl:4]: Let's mention the airToVac flag in the docstring, including its default value and that it is only pass along if the linelist_fn matches r"espresso\.mas$" or r"VALD".

Comment (eford) [reply src/pipeline/prep_line_list.jl:4]: Also convention is avoid capital letters for local variables like this.  Maybe rename "convert_air_to_vacuumb"?

**Comment by alexander-wise** [comment]
updated parameter names here and in EchelleCCFs to be consistent and in docstrings

**Review by eford — APPROVED** [review]
Thanks

**Pull request merged**